src/main/java/mate/academy/spring/dto/response/MovieSessionResponseDto.java

```diff
@@ -0,0 +1,54 @@
+package mate.academy.spring.dto.response;
+
+import java.time.LocalDateTime;
+import mate.academy.spring.model.CinemaHall;
+import mate.academy.spring.model.Movie;
+
+public class MovieSessionResponseDto {
+
+    private Long id;
+    private Movie movie;
+    private CinemaHall cinemaHall;
```

Comment:
```suggestion
    private Long movieId;
    private Long cinemaHallId;
```
need to return/search by id from ResponseDto's